spring/boot3-webflux-autoconfigure/src/test/java/com/linecorp/armeria/spring/web/reactive/ByteBufLeakTest.java

```diff
@@ -54,17 +47,16 @@
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
-@RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
-public class ByteBufLeakTest {
+@Timeout(10)
+class ByteBufLeakTest {
 
     private static final AtomicInteger completed = new AtomicInteger();
     private static final Queue<NettyDataBuffer> allocatedBuffers = new ConcurrentLinkedQueue<>();
 
     private static final AtomicBoolean requestReceived = new AtomicBoolean();
 
     @SpringBootApplication
-    @Configuration
```

Comment:
So now `@Configuration` isn't needed anymore? If so, could you also remove this annotation in other tests?

Reply:
`@SpringBootApplication` itself already contains `@Configuration`. So I thought it was enough to remove it.

`@SpringBootApplication`
   ㄴ `@SpringBootConfiguration`
       ㄴ `@Configuration`

Reply:
> So I thought it was enough to remove it.

Yes. How about also removing `@Configuration` in other classes? e.g.
https://github.com/line/armeria/pull/5036/files#diff-c1976a799fb6daafd188eecb992b2a31bb17d9147a5a1a324a565159b304fbb1R39
https://github.com/line/armeria/pull/5036/files#diff-27e240cb1631926a25e13a3c646e67aac657a603f3b4cf3be19b3a0e8e2e47faR60

Reply:
Oh, I was missing something. I'll edit 😉